### PR TITLE
Admin GUI dev tests and support for docker tests.

### PR DESCRIPTION
**Pull request by sumauppa**
Introduced docker parameter to run docker tests, when secure admin is already on.
Added few fixes for admin GUI dev tests.

### CODE REVIEW

**Comment by sumauppa** [comment]
@glassfishrobot Run CI tests please

**Comment by sumauppa** [comment]
@glassfishrobot Run CI tests please

**Comment by sumauppa** [comment]
@glassfishrobot Run CI tests please

**Comment by sumauppa** [comment]
@glassfishrobot Run CI tests please

**Comment by sumauppa** [comment]
@glassfishrobot Run CI tests please

**Comment by sumauppa** [comment]
@glassfishrobot Run CI tests please

File: main/appserver/admingui/devtests/src/test/java/org/glassfish/admingui/devtests/SpecificTestRule.java [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /*
  * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
  * 
- * Copyright (c) 2010-2011 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2010-2017 Oracle and/or its affiliates. All rights reserved.
```
Comment (arindam-bandyopadhyay) [inline main/appserver/admingui/devtests/src/test/java/org/glassfish/admingui/devtests/SpecificTestRule.java]: Please sync you fork with upstream master , The copyright template is wrong in your fork

**Review by arindam-bandyopadhyay — CHANGES_REQUESTED** [review]

**Comment by sumauppa** [comment]
@glassfishrobot Run CI tests please

File: main/appserver/admingui/devtests/src/test/java/org/glassfish/admingui/devtests/SpecificTestRule.java (hunk 1 above)
Comment (sumauppa) [inline main/appserver/admingui/devtests/src/test/java/org/glassfish/admingui/devtests/SpecificTestRule.java]: Done.

**Comment by sumauppa** [comment]
@glassfishrobot Run CI tests please

**Comment by sumauppa** [comment]
@glassfishrobot Run CI tests please

**Comment by sumauppa** [comment]
@glassfishrobot Run CI tests please

**Review by arindam-bandyopadhyay — APPROVED** [review]

**Pull request merged**